java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -240,11 +177,9 @@ static ColumnVector fromViewWithContiguousAllocation(long columnViewAddress, Dev
    */
   public ColumnVector toTitle() {
```

Comment:
I know that this would be cyclic dependency, but I thought that a `ColumnView` would have all of the high level processing APIs, like `toTitle` in it, so we could do things like get the first child of a struct and call `toTitle` directly on that, and the output would be a `ColumnVector`